config/database.yml

```diff
@@ -58,7 +58,9 @@ development:
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: expense_management_system_test
+  database: ems_test
+  username: postgres
+  password: 1234
```

Comment:
bad practice. Use `.env` for credentials.

Reply:
done & added .env_example